GOVERNANCE.md

```diff
@@ -12,6 +12,8 @@ ResBaz is committed to the following values:
 
 ## Governance
 
+***TODO: this is wrong!!!***
```

Comment:
Anyone have any ideas for how to revise this? I don't think the "board of directors" should exist, or make all the decisions

Reply:
agreed.